web/src/contexts/LinuxConfigContext.tsx

```diff
@@ -1,16 +1,6 @@
 import { createContext, useContext } from 'react';
```

Comment:
Both the `LinuxConfigContext` and the `KubernetesConfigContext` seem to exist for the sole need of having the admin console port later on (once the installation is finished). I think it would be worthwhile discussing cutting down this complexity and instead using React Query to get the config. My understanding from react query is that if we use consistent cache keys this wouldn't even result in a new request and instead re-use the initial data unless it's past the staleness ttl. Or worst case we get an extra cheap request with a fresh config instead of needing to use context for a couple of values.